doc/steam_license_readme.txt

```diff
@@ -0,0 +1,20 @@
+﻿Thrive Game
+Copyright (C) 2013-2021  Revolutionary Games
+
+This version of Thrive is distributed through Steam by Revolutionary Games Studio
```

Comment:
Is this a premade text? If not I would rewrite this a bit because this sounds a bit like Steam was made by us to me

Reply:
It doesn't sound like that to me but what would you suggest instead?.

Reply:
Maybe `This version of Thrive is distributed through Valve's digital distribution service Steam by Revolutionary Games Studio`

Reply:
Well that sounds pretty clunky, to be fair my version does as well, but I'll put this suggestion in.